pkg/occlient/occlient.go

```diff
@@ -49,6 +52,41 @@ var (
 	namespace   string
 )
 
+func init() {
```

Comment:
We shouldn't use init for this :-(
We will have to construct struct that will represent openshift client and that will hold all the `client-go` clients